CRM/ManualDirectDebit/Page/BatchList.php

```diff
@@ -159,15 +159,15 @@ private function buildSearchParams($typeIdInRequest) {
 
     switch (TRUE) {
       case !empty($createdDateFrom) && !empty($createdDateTo):
-        $param['created_date'] = ['BETWEEN' => [$createdDateFrom, $createdDateTo]];
+        $param['created_date'] = ['BETWEEN' => [$createdDateFrom . ' 00:00:00', $createdDateTo . ' 23:59:59']];
```

Comment:
@MiyaNoctem  should we have $createdDateFrom . ' 00:00:00' and $createdDateTo . ' 23:59:59' as new variables? as I see that they are repeated?